README.md

```diff
@@ -9,12 +9,25 @@ The `@cap-js/openapi` is a package that provides support for OpenAPI document co
 ### Table of Contents
 
 - [Requirements and Setup](#requirements-and-setup)
+- [Generate OpenAPI document](#generate-openapi-document)
 - [Contributing](#contributing)
 - [Code of Conduct](#code-of-conduct)
 - [Licensing](#licensing)
 
 ## Requirements and Setup
 
+### Installation
+
+```sh
+$ npm install @cap-js/openapi
+```
+
+### Usage
+
+```sh
+require("@cap-js/openapi").registerOpenapiCompileTarget()
+```
+
 ### Generate OpenAPI document
```

Comment:
You can remove one `#` from here to make it the sub-heading

Reply:
done